js/valid.js

```diff
@@ -0,0 +1,18 @@
+// Form Validation Code
+
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const errorText = document.querySelector('.error-text');
+
+/* the below function will check if the email entered is fully lowerCase */
+
+function emailCheckLowerCase(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    errorText.classList.remove('hide');
+    e.preventDefault();
+  } else {
+    errorText.classList.add('hide');
+  }
+}
+form.addEventListener('change', emailCheckLowerCase);
```

Comment:
- [ ] Kindly make sure your error message is only triggered when the user clicks on the submit button, currently, it appears if any changes happen in the form input fields and the email is not lowercase.

Reply:
rectified